docs/content/1.guide/6.fetch.md

```diff
@@ -0,0 +1,73 @@
+---
+icon: i-ri-global-line
+---
+
+# Fetch
+
+Nitro provides a built-in fetch API that can be used to get data from server endpoints or from other sources. It's build on top of the [unjs/ofetch](https://ofetch.unjs.io) package.
+
+This is important to use it instead of the built-in Node `fetch` or Axios because it's optimized for serverless environments and agnostic to the runtime.
+
+::alert{type="info"}
+We **do not overwrite** the built-in `fetch` function, so you can still use it if you want.
+::
+
+## Usage
+
+In you handler, you've just to call the `$fetch` function to make a request. Automatically, the response will be smartly parsed using [unjs/destr](https://destr.unjs.io).
+
+```ts [Router Handler]
+export default defineEventHandler(async (event) => {
+  const data = await $fetch('https://api.github.com/orgs/unjs/repos')
+
+  return data
+})
+```
+
+You can pass a generic type to the `$fetch` function to get a better type inference:
+
+```ts [Router Handler]
+import { Repo } from '~/types'
+
+export default defineEventHandler(async (event) => {
+  const data = await $fetch<Repo[]>('https://api.github.com/orgs/unjs/repos')
+
+  return data
+})
+```
+
+You can pass many options to the `$fetch` function like the method, headers, body, query, etc.
+
+```ts [Router Handler]
+import { Repo } from '~/types'
+
+export default defineEventHandler(async (event) => {
+  const data = await $fetch<Repo[]>('https://api.github.com/orgs/unjs/repos', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: {
+      name: 'my-repo'
+    }
+  })
+
+  return data
+})
+```
+
+See more about the usage of the `$fetch` function in the [unjs/ofetch](https://ofetch.unjs.io) documentation.
+
+## Internal Fetch
```

Comment:
```suggestion
## Direct Fetch
```

/cc @danielroe how do you like to call this functionality across docs?

Reply:
I think I like 'internal' because it suggests it's happening _within_ the nitro process. 'Direct' is less clear I think (what would an 'indirect' fetch mean?)

We could also brainstorm other ideas if you don't like 'internal'.

Reply:
Yeah but also Internal makes it sound it is framework internal while we are still talking about fetching user API endpoints. Maybe Fetching without base URL? or server fetch? :)

Reply:
yes im aware that currently in nuxt it is mentioned as internal…

local fetch is also good idea! (hiding more technical levels it is simply possibility to fetch without base url in server side if helps for brainstorming)

Reply:
How about 'in-app fetch'?

Reply:
The app sounds a little cryptic (also in Nitro terms App=Server but in Nuxt terms could be more confusing if we use same _App_ Term).

Reply:
`in-server fetch`?

Reply:
I like it :)